CHANGELOG.md

```diff
@@ -6,6 +6,55 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 in the naming of release branches.
 
+
```

Comment:
did you mean to put all this under the `## [Unreleased]` section below?

Reply:
Yeah probably got deleted accidentally. I'll add it back with the next change I have to do

Reply:
Oh, I see, I didn't want to clamp all those changes in with the current unreleased. 

I am thinking it might be a good idea to have this whole changelog as a separate section in the new "unreleased version". What do you think? Eg.

```
## [Unreleased]
### Refactor #2901, #.29..
#### Added
...
#### Changed

### Added
-  Added `coinsPerUTxOByteToCoinsPerUTxOWord` helper function for Babbage
   #2896
### Changed
...
```

We can of course merge it into the usual format, but I am just thinking that the distinction/separation might be useful to the reader. This is just an idea, I don't have a strong opinion on this

Reply:
hmm, I don't have a strong opinion either. I was just adhering blindly to "the rules", but perhaps this is exception worthy.